payments/credit-card/index.md

```diff
@@ -0,0 +1,514 @@
+---
+title: Swedbank Pay Payments Credit Card
+sidebar:
+  navigation:
+  - title: Payments
+    items:
+    - url: /payments/
+      title: Introduction
+    - url: /payments/credit-account
+      title: Credit Account Payments
+    - url: /payments/credit-card
+      title: Credit Card Payments
+    - url: /payments/invoice
+      title: Invoice Payments
+    - url: /payments/direct-debit
+      title: Direct Debit Payments
+    - url: /payments/mobile-pay
+      title: Mobile Pay Payments
+    - url: /payments/swish
+      title: Swish Payments
+    - url: /payments/vipps
+      title: Vipps Payments
+---
+
+{% include alert.html type="warning"
+                      icon="warning"
+                      header="Site under development"
+                      body="The Developer Portal is under construction and should not be used to integrate against Swedbank Pay's APIs yet." %}
+
+
+>The basic redirect purchase scenario is the most common way to implement card payments.
+
+# Card Payment Pages
```

Comment:
The first-level heading is reserved for the `title`, so all `.md` files should start at header level two (`##`). That being said, this entire header is just a repetition of the `title` so it can just be removed.

```suggestion
```